src/groups/group.jl

```diff
@@ -237,7 +237,7 @@ end
     inv(G::AbstractGroupManifold, x)
 
 Inverse $x^{-1} ∈ G$ of an element $x ∈ G$, such that
-$x \circ x^{-1} = x^{-1} \circ x = e ∈ G$.
+$x ◦ x^{-1} = x^{-1} ◦ x = e ∈ G$.
```

Comment:
I tried unicode here earlier, but it ended up looking really cramped. See https://julianlsolvers.github.io/Manifolds.jl/previews/PR104/manifolds/group.html#Base.inv-Tuple{Manifold,Any}

Reply:
I have to check, I think there is a middle-circle-symbol that is not as small as the celsius circle thingy.